runtime/vm/jvminit.c

```diff
@@ -1994,9 +1991,34 @@ VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved)
 			}
 #endif
 
+			IDATA xtuneQuickstartIndex =  FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_TUNE_QUICKSTART, NULL);
+			IDATA xquickstartIndex =  FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XQUICKSTART, NULL);
+			IDATA clientIndex =  FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_CLIENT, NULL);
+			/* VMOPT_TUNE_QUICKSTART, VMOPT_XQUICKSTART and VMOPT_CLIENT are
+			 * aliases, so take the largest index of these 3 */
+			if (xquickstartIndex > xtuneQuickstartIndex) {
+				xtuneQuickstartIndex = xquickstartIndex;
+			}
+			if (clientIndex > xtuneQuickstartIndex) {
+				xtuneQuickstartIndex = clientIndex;
+			}
```

Comment:
It's easier to see that the names being compared are the same used in the assignment if they appear in the same order:
```
	if (xtuneQuickstartIndex < xquickstartIndex) {
		xtuneQuickstartIndex = xquickstartIndex;
	}
	if (xtuneQuickstartIndex < clientIndex) {
		xtuneQuickstartIndex = clientIndex;
	}
```

Reply:
Done.